PyKCS11/__init__.py

```diff
@@ -521,10 +507,42 @@ def load(self, pkcs11dll_filename=None, *init_string):
                     "ref": self.lib,
                     "nb_users": 0
                     }
+            
+        # remember the lib file name
+        self.pkcs11dll_filename = pkcs11dll_filename
 
         # increase user number
         PyKCS11Lib._loaded_libs[pkcs11dll_filename]["nb_users"] += 1
 
+        return self
+
+    def unload(self):
+        """
+        unload the current instance of a PKCS#11 library
+        """
+
+        # in case NO library was found and used
+        if not hasattr(self, "pkcs11dll_filename"):
+            return
+
+        if self.pkcs11dll_filename not in PyKCS11Lib._loaded_libs:
```

Comment:
If you ask me about slightly decreased coverage -- this is the line that remains uncovered. This condition is not supposed to be met: just an additional consistency check. I can convert it to an assertion or just remove it, if you like.